lib/open_api_spex/open_api/map.ex

```diff
@@ -0,0 +1,395 @@
+defmodule OpenApiSpex.OpenApi.Map do
+  @moduledoc """
```

Comment:
This module should be an implementation detail without a moduledoc. The entry point should be in `OpenApiSpex` or `OpenApiSpex.OpenApi`

Reply:
Agreed!